streamparse/cli/submit.py

```diff
@@ -205,6 +205,17 @@ def submit_topology(name=None, env_name="prod", workers=2, ackers=2,
             if isinstance(inner_shell, ShellComponent):
                 if 'streamparse_run' in inner_shell.execution_command:
                     inner_shell.execution_command = streamparse_run_path
+                    
+    # Additional options
+    additional_options = env_config.get('options', None)
+    if additional_options and isinstance(additional_options, dict):
+        if isinstance(options, dict):
+            additional_options.update(options)
+        options = additional_options
+    if not workers:
```

Comment:
Oh, a brief explanation:
In our custom fork of this file, I've set defaults of the method to None, to be able check if something was passed or not (since there could be passed 2 as well). But it turned out, that it is set to 2, even though the defaults are None. 
I wanted to leave precedence to CLI arguments before config.json options, but it turned out, that there is always passed number 2 to this method (from outside), even with no -p  (or similar option) used. 
--
So I'm trying to find a way to ensure that defaults < config.json < CLI args.
Which I was not able to do so far.